### DISCUSSION TITLE
ADT won't load the key

### DISCUSSION THREAD

**Discussion by Yaniko** [airsdk/Adobe-Runtime-Support#637] [Q&A]
Hi.
```
adt -checkstore -storetype PKCS12 -keystore C:\ios.p12 -storepass VhA9#H8GINf" -provisioning-profile C:\pro.mobileprovision
password:
could not load keystore file (password may be incorrect)
```
IDEA accepts that password, when it starts to package application. After that IDEA fails to package, when it calls adt package.

I added the key to windows passwords. After that adt managed to read it 
(with ` -keystore Windows-MY -alias .... ` )
I made a key with a new password. It worked fine. I changed its password back to the first one. ADT failed.
So, I suppose the problem is the exact password. Maybe because it contains a quote symbol?
adt version: 32.0.0.116, 31.0.0.96

Forgot to mention: it works fine on a mac, but fails on windows

**Comment by ajwfrost** [comment]
Yes I suspect the quote mark is a problem on Windows; have you tried escaping it? We can check though as it may be the interactions between the various batch and Java applications causing the problem..

**Comment by Yaniko** [comment]
I've just tried `-storepass VhA9#H8GINf\"`. It didn't work either.
Also I can't escape a quote mark when it prompts for a password and I type it ?

**Comment by Oldes** [comment]
@Yaniko what about using different password.. or try this: https://stackoverflow.com/a/7760618/494472

**Comment by Yaniko** [comment]
Yeah, changing password helps, I wrote about that.

I spent a couple of hours trying to figure out what had happened to my PC before I realised it was the password. I want to warn others.
The problem is, I use IDEA to package applications. It just prompts for password and nothing else. That's why all escaping-type workarounds won't help.
